apps/mobile/app/components/LabelItem.tsx

```diff
@@ -2,20 +2,23 @@
 /* eslint-disable react-native/no-inline-styles */
 import React, { FC, ReactNode } from "react"
 import { View, StyleSheet, Text } from "react-native"
-import { typography, useAppTheme } from "../theme"
+import { typography } from "../theme"
 
 interface Props {
 	label: string
 	background: string
 	icon: ReactNode | undefined
 }
 const LabelItem: FC<Props> = ({ label, background, icon }) => {
-	const { colors } = useAppTheme()
+	const words = label.split(" ")
+	const formattedLabel = words.map((word) => word.charAt(0).toUpperCase() + word.slice(1)).join(" ")
```

Comment:
@desperado1802 instead of doing this logic can't we just use `text-transform: capitalize` CSS property?